services/121-service/src/events/events.service.ts

```diff
@@ -339,7 +339,9 @@ export class EventsService {
       'programId',
       'registrationCreated',
       'registrationCreatedDate',
-      'financialServiceProvider',
+      'financialServiceProviderName',
```

Comment:
Not entirely sure what you mean here. This is a list of strings of registrationview properties not to log in the event service.

I added the financialServiceProviderName to the registrationViewEntity, because I though we would maybe need it in the frontend when changing attributes in the fronted to find out which ones are required for that fsp.